frontend/src/routes/ClusterManagement/Clusters/components/ClusterPolicySidebar.tsx

```diff
@@ -131,14 +184,84 @@ function DetailsView(props: {
     const [tabState, setTabState] = useState<React.ReactText>(0)
     const classes = useStyles()
 
+    const onSelect = (id: string) => {
+        switch (id) {
+            case 'action 1':
+                console.log('perform action 1')
+                break
+            case 'action 2':
+                console.log('perform action 2')
+                break
+        }
+    }
+
+    function matchedDate() {
+        let d = new Date(_.get(selectedPolicy, 'results[0].data.created_at', '')).toDateString()
+        return d
+    }
+
+    function categories() {
+        let categories = _.get(selectedPolicy, 'results[0].category', '')
+        if (categories && categories !== '') {
+            const categoriesToHide = categories.slice(1)
+            return <AcmLabels labels={categories.split(',')} collapse={categoriesToHide} />
+        }
+    }
+
     return (
-        <div className={classes.sidebarBody}>
-            <div className={classes.sidebarTitleText}>
-                <button onClick={() => setDetailsView(false)} className={classes.backAction}>
-                    {t('policy.report.flyout.back')}
-                </button>
+        <div className={classes.body}>
+            <div className={classes.backAction}>
+                <AngleLeftIcon />
+                <button onClick={() => setDetailsView(false)}>{t('policy.report.flyout.back')}</button>
+            </div>
+            <div className={classes.detailTitleGroup}>
+                <div className={classes.titleText}>{_.get(selectedPolicy, 'results[0].message', '')}</div>
+                <div className={classes.detailActions}>
```

Comment:
For the current iteration we don't have any actions to take on the Insight. In the future we will want to be able to press a button and remediate the vulnerability. But for now we can remove the actions dropdown.